templates/request/get/get.handlebars

```diff
@@ -72,3 +72,67 @@
         done();
       });
     });
+{{#if isLoadTest}}
+    it('load tests with {{length description}}', function(done) {
+      arete.loadTest({
+        name: 'PUT THE NAME HERE',
+        requests: {{requests}},
+        concurrentRequests: {{concurrent}},
+        targetFunction: function(callback) {
+          request({
+            url: '{{pathify path}}',
+            {{#ifCond queryParameters queryApiKey}}
+            qs: {
+              {{#if queryApiKey}}{{queryApiKey.type}}: process.env.{{queryApiKey.name}}{{#if queryParameters}},
+              {{/if}}{{/if}}{{#if queryParameters}}{{#each queryParameters}}{{this.name}}: 'DATA GOES HERE'{{#unless @last}},{{/unless}}{{/each}}{{/if}}
+            },
+            {{/ifCond}}
+            method: 'GET',
+            headers: {
+              'Content-Type': '{{contentType}}'{{#if headerParameters}},
+              'Custom-Header': { {{#each headerParameters}}
```

Comment:
Oh!!!!!! I understand what you mean! Soorrry~~Yes, this is a bug...The Custom-header should be the parameter name......I misunderstood the request module doc, I thought "Custom-header" was a object name involving all custom headers, so all of the json list should be the value, but actually this was just an example how custom-headers can be set....